reflex/components/radix/themes/components/checkbox.py

```diff
@@ -62,9 +66,21 @@ def get_event_triggers(self) -> Dict[str, Any]:
         """
         return {
             **super().get_event_triggers(),
-            "on_checked_change": lambda e0: [e0],
+            # "on_change": lambda e0: [e0],
+            EventTriggers.ON_CHANGE: lambda e0: [e0],
+            # EventTriggers.ON_CHECKED_CHANGE: lambda e0: [e0],
         }
 
+    # class level attrbute of the names that you want to replace
+
+    def render(self) -> dict:
+        """Render the component."""
+        render_dict = super().render()
+        print(render_dict)
+        self._replace_prop_names(render_dict)
+        print(render_dict)
+        return render_dict
```

Comment:
i think this logic should go in `Component.render` itself. since the `_rename_props` attribute is defined on `Component`, it wouldn't really make sense to be able to set `_rename_props`, but then not have it work until the downstream component overrides `render`.

Reply:
added